### PR TITLE
Feature: playlists

### PR DESCRIPTION
**Pull request by jessopb**

### CODE REVIEW

File: ui/component/autoplayCountdown/index.js [hunk 1]
```diff
@@ -9,10 +9,28 @@ import { selectModal } from 'redux/selectors/app';
 
 /*
 AutoplayCountdown does not fetch it's own next content to play, it relies on <RecommendedContent> being rendered. This is dumb but I'm just the guy who noticed
+
+get collectionId and collectionIndex from content to use in collectoinSelector
+
  */
 const select = (state, props) => {
-  const nextRecommendedUri = makeSelectNextUnplayedRecommended(props.uri)(state);
+  const { history, location } = props;
```
Comment (infinite-persistence) [inline ui/component/autoplayCountdown/index.js:17]: Minor:  unused var

File: ui/page/collection/view.jsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+import FileDescription from 'component/fileDescription';
+import { COLLECTIONS_CONSTS } from 'lbry-redux';
+import Icon from 'component/common/icon';
+import * as ICONS from 'constants/icons';
+
+export const PAGE_VIEW_QUERY = 'view';
+export const PUBLISH_PAGE = 'publish';
+export const EDIT_PAGE = 'edit';
+export const ADVANCED_EDIT_PAGE = 'advanced';
+
```
Comment (infinite-persistence) [inline ui/page/collection/view.jsx:21]: minor: unused vars

File: ui/component/collectionPreviewOverlay/view.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+type Props = {
+  uri: string,
+  collectionId: string,
+  collectionName: string,
+  collectionCount: number,
+  editedCollection?: Collection,
+  pendingCollection?: Collection,
+  claim: ?Claim,
+  collectionItemUrls: Array<string>,
+  collectionId: string,
```
Comment (infinite-persistence) [inline ui/component/collectionPreviewOverlay/view.jsx:15]: duplicate `collectionId`

File: ui/component/collectionEdit/view.jsx [hunk 4]
```diff
@@ -0,0 +1,437 @@
...
+      collectionParams['title'] = title;
+    }
+
+    return collectionParams;
+  }
+
+  React.useEffect(() => {
+    const collectionClaimIds = JSON.parse(collectionClaimIdsString);
+    setParams({ ...params, claims: collectionClaimIds });
+  }, [collectionClaimIdsString, setParams]);
```
Comment (infinite-persistence) [inline ui/component/collectionEdit/view.jsx:166]: Should depend on `params` too?  Wondering if this is causing the Anonymous problem.

**Review by infinite-persistence — COMMENTED** [review]
- https://workflowy.com/s/pr-6157-playlist-des/qUMXmxHb2LmhTp1F (feel free to edit)
- See inline comments.

File: ui/component/autoplayCountdown/index.js [hunk 5]
```diff
@@ -9,10 +9,28 @@ import { selectModal } from 'redux/selectors/app';
...
+  const { search } = location;
+  const urlParams = new URLSearchParams(search);
+  const collectionId = urlParams.get('pl');
+  const collectionIndex = urlParams.get('plindex');
+
```
[the remark this answers is not in the document]
Comment (jessopb) [reply ui/component/autoplayCountdown/index.js:22]: Yes, just fixed this!

File: ui/modal/modalRemoveFile/view.jsx [hunk 6]
```diff
@@ -29,7 +29,7 @@ function ModalRemoveFile(props: Props) {
   return (
     <Modal isOpen contentLabel={__('Confirm File Remove')} type="card" onAborted={closeModal}>
       <Card
-        title={__('Remove File')}
+        title={__('Remove Collection')}
```
[the remark this answers is not in the document]
Comment (jessopb) [reply ui/modal/modalRemoveFile/view.jsx:32]: no...

**Review by infinite-persistence — COMMENTED** [review]
- [ ] The value seems to be stuck
  - image

File: ui/component/collectionEdit/view.jsx (hunk 4 above)
Comment (jessopb) [reply ui/component/collectionEdit/view.jsx:166]: anonymous problem was partly because I wasn't reading 'incognito' prop.

**Comment by iamflorencejay** [comment]
LETS GOOOOO~

**Pull request merged**